src/Verb.js

```diff
@@ -97,7 +97,7 @@ TinCan client library
             if (typeof cfg === "string") {
                 this.id = cfg;
                 this.display = {
-                    und: this.id
+                    en: this.id
```

Comment:
This is what they want to use for the 'id' and we can't know whether it is English as it is an IRI (or the special 0.9 case where we do know, see additional comments). Now it could be argued that in that case we shouldn't be setting a 'display' property at all.